src/Aspire.Hosting/Dashboard/ResourceCollection.cs

```diff
@@ -37,8 +27,8 @@ public ResourceSubscription Subscribe()
             ImmutableInterlocked.Update(ref _outgoingChannels, static (set, channel) => set.Add(channel), channel);
 
             return new ResourceSubscription(
-                Snapshot: _snapshot.Values.ToList(),
-                Subscription: new ChangeEnumerable(channel, RemoveChannel));
+                Snapshot: [.. _snapshot.Values],
```

Comment:
Maybe it is just me, but I prefer ye olde `ToList()`.

Reply:
I'm happy either way. My preference is that whatever's decided is applied in `.editorconfig` so that we don't get suggestions in the IDE that we don't want, as ignoring them takes a small amount of effort.